tests/src/test/scala/whisk/core/cli/test/WskEntitlementTests.scala

```diff
@@ -35,17 +33,22 @@ import whisk.core.entity.Subject
 import whisk.core.entity.WhiskPackage
 
 @RunWith(classOf[JUnitRunner])
-class WskEntitlementTests extends TestHelpers with WskTestHelpers with BeforeAndAfterAll {
+abstract class WskEntitlementTests extends TestHelpers with WskTestHelpers with BeforeAndAfterAll {
 
-  val wsk = new Wsk
+  val wsk: BaseWsk
   lazy val defaultWskProps = WskProps()
   lazy val guestWskProps = getAdditionalTestSubject(Subject().asString)
 
   override def afterAll() = {
     disposeAdditionalTestSubject(guestWskProps.namespace)
   }
 
+  def getForbiddenCode(): Int
+  def getTimeoutCode(): Int
+  def getNotfoundCode(): Int
```

Comment:
these could be abstract values instead of methods.

Reply:
also you can drop the `get` prefix which is more scala idiomatic.

Reply:
Done.